### PR TITLE
chore: increase vacuum batch size

### PR DESCRIPTION
**Pull request by zwang28**
I hereby agree to the terms of the [Singularity Data, Inc. Contributor License Agreement](https://gist.github.com/skyzh/0663682a70b0edde7ae991492f2314cb#file-s9y_cla).

## What's changed and what's your intention?

As we've support batch delete for object store, we can increase batch size of each vacuum task, so that to reduce overhead of small tasks.

## Checklist

- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

## Documentation

If your pull request contains user-facing changes, please specify the types of the changes, and create a release note. Otherwise, please feel free to remove this section.

### Types of user-facing changes

Please keep the types that apply to your changes, and remove those that do not apply.

* Installation and deployment 
* Connector (sources & sinks)
* SQL commands, functions, and operators
* RisingWave cluster configuration changes
* Other (please specify in the release note below)

### Release note

Please create a release note for your changes. In the release note, focus on the impact on users, and mention the environment or conditions where the impact may occur.

### CODE REVIEW

File: src/meta/src/hummock/vacuum.rs [hunk 1]
```diff
@@ -103,7 +103,7 @@ where
 
         // Dispatch the vacuum task
         let mut batch_idx = 0;
-        let batch_size = 32usize;
+        let batch_size = 500usize;
```
Comment (Li0k) [inline src/meta/src/hummock/vacuum.rs:106]: AWS restricts the number of objects per request to 1000, refer to `delete_objects` function of s3.rs, are they the same?
```rust
const MAX_LEN: usize = 1000;
```

Comment (zwang28) [reply src/meta/src/hummock/vacuum.rs:106]: Yes. And the delete_objects has already split large input into multiple s3 calls. So it's OK to set arbitrary batch_size here.

**Review by Li0k — APPROVED** [review]
LGTM

**Comment by ALeitert** [comment]
LGTM!

**Review by ALeitert — APPROVED** [review]

**Pull request merged**